model/jpa/src/main/java/org/keycloak/connections/jpa/updater/liquibase/LiquibaseJpaUpdaterProvider.java

```diff
@@ -152,8 +153,8 @@ protected void updateChangeSet(Liquibase liquibase, Connection connection, Write
                 statementsToExecute.add(new SetNullableStatement(database.getLiquibaseCatalogName(), database.getLiquibaseSchemaName(),
                         changelogTable.getName(), DEPLOYMENT_ID_COLUMN, "VARCHAR(10)", false));
 
-                ExecutorService executorService = ExecutorService.getInstance();
-                Executor executor = executorService.getExecutor(liquibase.getDatabase());
+                ExecutorService executorService = Scope.getCurrentScope().getSingleton(ExecutorService.class);
+                Executor executor = executorService.getExecutor("jdbc", liquibase.getDatabase());
```

Comment:
This is because `getExecutor(Database)` is deprecated and the API says we should use this version that takes the executor name, in our case `jdbc`. I can for sure turn this into a constant.